client/me/account-close/main.jsx

```diff
@@ -108,7 +127,16 @@ class AccountSettingsClose extends Component {
 									<ActionPanelFigureListItem>
 										{ translate( 'Personal details' ) }
 									</ActionPanelFigureListItem>
-									<ActionPanelFigureListItem>{ translate( 'Sites' ) }</ActionPanelFigureListItem>
+									<ActionPanelFigureListItem className="account-close__sites-item">
+										{ translate( 'Sites' ) }
+										<Gridicon size={ 18 } onClick={ this.handleSiteDropdown } icon="chevron-down" />
+									</ActionPanelFigureListItem>
+									{ this.state.showSiteDropdown && (
+										<SiteSelector
```

Comment:
I think this should be inside the `ActionPanelFigureListItem` for the markup to validate:

<img width="584" alt="Screen Shot 2019-11-14 at 16 12 54" src="https://user-images.githubusercontent.com/17325/68823775-a6021880-06f9-11ea-80e6-d3a86c658f18.png">

Reply:
Will fix when I add a commit to remove the icon! (Otherwise the icon looks a bit odd)

<img width="216" alt="Screenshot 2019-11-14 at 20 55 12" src="https://user-images.githubusercontent.com/43215253/68895400-18d2ba00-0721-11ea-8d8c-bb91884747e3.png">

Reply:
Done! :)